app/controllers/api/mentor_discussion_posts_controller.rb

```diff
@@ -35,6 +35,20 @@ def create
       head 200
     end
 
+    def update
+      post = Solution::MentorDiscussionPost.find_by(uuid: params[:id])
+
+      return render_404(:mentor_discussion_post_not_found) if post.blank?
+      return render_403(:permission_denied) unless post.author == current_user
+
+      if post.update(content_markdown: params[:content])
+        DiscussionPostListChannel.notify!(post.discussion, post.iteration)
+        render json: SerializeMentorDiscussionPost.(post, current_user)
+      else
+        render_failed_validations(post.errors)
```

Comment:
@iHiD I don't know how you want to structure this, but it's a bit tricky since we want to pass arguments to the `I18n.t` call. I just did it this way to make the tests pass, but I'll leave you to it to structure this in the way you see correct.

Reply:
I'll do this tomorrow if you've not done it first :)